models/surveySubmissions.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    name: "surveySubmissions",
+    schema: {
+      surveyId: "ObjectId",
+      name: String,
+      createdBy: String,
+      status: String,
+      evidencesStatus: Array,
+      evidences: Object,
+      criteria: Array,
+      themes: Array,
+      answers: Object,
+      generalQuestions: Object,
+      completedDate: Date,
+      solutionId: "ObjectId",
+      solutionExternalId: String,
+      programId: "ObjectId",
+      programExternalId: String,
+      submissionNumber: Number,
+      isRubricDriven: { type : Boolean, default: false },
```

Comment:
@deepakr123 - Survey information is missing. Also, isAPrivateProgram flag is missing.

Reply:
resolved